app/models/academic_year.rb

```diff
@@ -10,4 +10,10 @@
 #  updated_at :datetime         not null
 #
 class AcademicYear < ApplicationRecord
+  has_many :claim_windows, class_name: "Claims::ClaimWindow"
+  has_many :claims, through: :claim_windows
```

Comment:
Can we avoid leaking the claims-specific logic into the top-level `AcademicYear`, please? 🙏

Reply:
Similar to my other comment:
https://github.com/DFE-Digital/itt-mentor-services/pull/891/files#r1702024680

I would assume you would want the reverse of `belongs_to :academic_year` in ClaimWindow, in AcademicYear. Generally this is what is done when creating associations.

Reply:
@dp-daly We've discussed this with @Nitemaeric and have come to an agreement to remove the associations work from this PR. The reasoning is to implement them if/when the associations are required to be used as part of the code.